packages/@ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -1056,9 +1056,12 @@ export class AngularCompilerPlugin {
         if (locales.indexOf(parentLocale) !== -1) {
           locale = parentLocale;
         } else {
-          throw new Error(
-            `Unable to load the locale data file "@angular/common/locales/${locale}", ` +
-            `please check that "${locale}" is a valid locale id.`);
+          this._warnings.push(`AngularCompilerPlugin: Unable to load the locale data file ` +
+            `"@angular/common/locales/${locale}", ` +
+            `please check that "${locale}" is a valid locale id.
+            Proceeding with default locale data registration i.e. 'en'.
```

Comment:
you can remove `Proceeding with default locale data registration i.e. 'en'.` please

Reply:
Done.

Reply:
@ocombe  I am done  with all the  changes, including test case modification.